service/history/historyEngine.go

```diff
@@ -766,7 +765,12 @@ func (e *historyEngineImpl) DescribeWorkflowExecution(
 	if executionInfo.HasRetryPolicy && (executionInfo.Attempt > 0) {
```

Comment:
You can also delete this part. The FirstDecisionTaskBackoffSeconds covers both retry and cron.

Reply:
done